pkgs/applications/terminal-emulators/wezterm/default.nix

```
@@ -97,6 +98,10 @@ rustPlatform.buildRustPackage rec {
     ln -s $out/bin/{wezterm,wezterm-mux-server,wezterm-gui,strip-ansi-escapes} "$OUT_APP"
   '';
 
+  passthru.tests = lib.optionalAttrs stdenv.isLinux {
```

```suggestion
  passthru.tests = {
```
wezterm is marked broken on darwin. We shouldn't need to duplicate this. It is fine if it fails for darwin.

---

I am hoping someone will eventually fix it, why would you explicitly want to not filter out a failing test?

---

I don' think we filter out tests anywhere else like this and darwin can't run them in general anyway.